bounties/npm/phpjs/1/vulnerability.json

```diff
@@ -0,0 +1,48 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-20",
+	"AffectedVersionRange": "*",
+	"Summary": "Prototype Pollution",
+	"Author": {
```

Comment:
@d3m0n-r00t - this should follow the format shown on https://github.com/418sec/huntr/tree/master/bounties.

Can you swap out the `Author` object for `Contributor` and insert your GitHub name into the `Discloser` section?

Reply:
@JamieSlome. I have added the requested changes. Sorry I didnt note the change. I was following the previous template.